README.md

```diff
@@ -18,7 +18,15 @@ injecting your own 802.11 frames, combining technics (VLAN hopping+ARP
 cache poisoning, VOIP decoding on WEP encrypted channel, ...),
 etc.
 
-See
+# Installation #
+
+Scapy requires a few depedencies depending of your OS to work correctly. It currently supports Unix-like operating systems (Linux, BSD, Mac OS X), and Windows.
```

Comment:
I will rather reformulate this sentence as:
"Scapy works without any external Python modules on Linux and BSD like operating systems. On Windows, you need to install some mandatory dependencies as described in the [documentation](http://scapy.readthedocs.io/en/latest/installation.html).

To benefit from all Scapy features, you might want to install Python modules, such as matplotlib or pycrypto."